angular.json

```diff
@@ -146,5 +146,8 @@
       }
     }
   },
-  "defaultProject": "Shop"
-}
+  "defaultProject": "Shop",
+  "cli": {
+    "analytics": false
+  }
+}
```

Comment:
GitHub needs empty line at the end of each file you want to add.

Reply:
Add